resources/js/screens/User.vue

```diff
@@ -137,6 +161,34 @@
         }
       },
 
+      async loadRoles() {
+        try {
+          const {data} = await axios.get(`/web/roles`)
+          this.roles    = data
+        } catch (e) {
+          console.error(e)
+          this.$notify({
+            text: 'Unable to process this request. Please try refreshing the page.',
+            type: 'error',
+          })
+        }
+      },
+
+      async setRole() {
+        try {
+          const id      = this.$route.params.id
+          await axios.patch(`/web/user/${id}/role`, {
+            role: this.userRole,
+          })
```

Comment:
Let's tell the user that something has changed.
```suggestion
          })
          
          this.$notify({
              text: 'Role updated successfully.',
              type: 'success'
          })
```